Gemfile

```diff
@@ -51,7 +51,7 @@ gem 'aker_permission_gem', github: 'sanger/aker-permission'
 
 gem 'aker-study-client', github: 'sanger/aker-study-client'
 gem 'aker-set-client', github: 'sanger/aker-set-client'
-gem 'matcon_client', github: 'sanger/aker-matcon-client'
+gem 'matcon_client', github: 'emrojo/aker-matcon-client', branch: 'bugfix_remote_slash_from_material_url'
```

Comment:
These changes have been merged in now, so please could you point this to the Sanger repo?